test/jdk/java/awt/image/MultiResolutionImage/MultiResolutionImageObserverTest.java

```diff
@@ -70,7 +70,7 @@ private static void waitForImageLoading(LoadImageObserver observer,
         long endTime = System.currentTimeMillis() + TIMEOUT;
 
         while (!observer.loaded && System.currentTimeMillis() < endTime) {
-            Thread.sleep(TIMEOUT / 10);
+            Thread.sleep(TIMEOUT / 100);
         }
 
         if (!observer.loaded) {
```

Comment:
The "loaded" field is read by the main thread but set by the image loader thread, probably it is better to mark it as volatile.